aws-spi-pekko-http/src/test/scala/org/apache/pekko/stream/connectors/awsspi/PekkoHttpClientSpec.scala

```diff
@@ -17,14 +17,21 @@
 
 package org.apache.pekko.stream.connectors.awsspi
 
-import java.util.Collections
+import com.typesafe.config.ConfigFactory
 
+import java.util.Collections
```

Comment:
can you move this import back to a separate group before the main body of imports?